docs/examples/http_exceptions.py

```diff
@@ -0,0 +1,53 @@
+import json
+
+from nameko.web.handlers import HttpRequestHandler
+from werkzeug.wrappers import Response
+
+
+class ApplicationError(Exception):
+    pass
+
+
+class HttpError(Exception):
+
+    def __init__(self, message, error, status_code):
+        self.payload = {
+            'error': error,
+            'message': message
+        }
+        self.status_code = status_code
+        super(HttpError, self).__init__(self.payload)
```

Comment:
It seems unconventional to pass a dict into ``Exception.__init__``. I think
```python
super(HttpError, self).__init__("{}: {}".format(error, message))
```
would be more normal. @davidszotten do you have an opinion on this?